meta-balena-common/recipes-bsp/grub/grub-bootconf_1.00.bb

```diff
@@ -0,0 +1,32 @@
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://${COREBASE}/meta/COPYING.MIT;md5=3da9cfbcb788c80a0384361b4de20420"
+SUMMARY = "Basic grub.cfg for use in EFI systems"
+DESCRIPTION = "Grub might require different configuration file for \
+different machines."
+HOMEPAGE = "https://www.gnu.org/software/grub/manual/grub/grub.html#Configuration"
+
+RPROVIDES:${PN} += "virtual-grub-bootconf"
+
+inherit grub-efi-cfg
+
+require conf/image-uefi.conf
+
+S = "${WORKDIR}"
+
+GRUB_CFG = "${S}/grub-bootconf"
+LABELS = "boot"
+
+ROOT ?= "root=/dev/sda2"
```

Comment:
what's this `ROOT` used for, and why default it to `/dev/sda2`?

Reply:
@alexgg No idea, I just copied the recipes from poky and updated them to use the newer 2.12 sources. This line is directly from the poky recipe.

Reply:
@jakogut maybe append the commit message to mention the Poky commit the recipe comes from for future reference?